### PR TITLE
Fix: Implement PDF preview inside dashboards

### PR DESCRIPTION
**Pull request by mumairr**
## Description

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [x] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

Dashboards use React Quill Editor

**What is the new behavior?**

Implements Draftjs based editor for dashboards (Text) to allow embedding of any external link, document on the widget.
Fixes up scroll issue on widgets created from map

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

## Other useful information
This PR is for backport based on version 2022.02.xx for use in GN. Same for the master has already been implemented last week,
 
https://github.com/geosolutions-it/C273-ISPRA-2023-UNEP-MAP/issues/42

### CODE REVIEW

File: web/client/themes/default/less/wizard.less [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
 
 .ms-wizard {
-    position: absolute;
+    // position: absolute;
```
Comment (dsuren1) [inline web/client/themes/default/less/wizard.less:33]: I don't think this is needed. Kindly check and revert this change

**Review by dsuren1 — CHANGES_REQUESTED** [review]
@mumairr 
Kindly amend these minor style issues. Thanks!
image

File: web/client/themes/default/less/wizard.less (hunk 1 above)
Comment (mumairr) [reply web/client/themes/default/less/wizard.less:33]: @dsuren1 this was needed to solve a small bug, it came into limelight after Ellano's comment #2 on my previous PR [here](https://github.com/geosolutions-it/MapStore2/pull/9680#issuecomment-1798226903)
Interestingly this came into play as well for this backport. If we add up a long text, links etc. in editor for geostories via map, scrollbar doesn't appear. Kindly check it

**Comment by mumairr** [comment]
> @mumairr Kindly amend these minor style issues. Thanks! image

Fixing styles for this one

**Comment by dsuren1** [comment]
@mumairr 
Looks like there is an open PR of the original issue with some fixes. Hence this PR https://github.com/geosolutions-it/MapStore2/pull/9696 has be accepted and tested in **DEV** before you can make any backport of this feature to `2022.02.xx`

File: web/client/themes/default/less/wizard.less (hunk 1 above)
Comment (mumairr) [reply web/client/themes/default/less/wizard.less:33]: It occurs only if you add dashboard txt widget via map for a geostory. I was also confused with it and Stefano guided me how to reproduce it.

image
image

**Comment by allyoucanmap** [comment]
Closing because has been solved here https://github.com/geosolutions-it/MapStore2/pull/9724
